src/debugging.ts

```diff
@@ -31,5 +31,5 @@ export function logAccountsState(accounts: ReadonlyArray<BcpAccount>): void {
 export function logSendJob(signer: MultiChainSigner, job: SendJob): void {
   const from = identityToAddress(signer, job.sender);
   const to = job.recipient;
-  console.log(`Sending ${job.wholeAmount} ${job.tokenTicker} from ${from} to ${to} ...`);
+  console.log(`Sending ${job.amount} ${job.tokenTicker} from ${from} to ${to} ...`);
```

Comment:
`${job.amount}` is `Amount` now, which should get a proper string representation. You can reuse `debugCoin`:

```
function debugCoin(coin: Amount): string {
   // ...
}
```

Since every `BcpCoin` is an `Amount`, you only need this one method.